src/main.c

```diff
@@ -299,6 +368,7 @@ int main(int argc, char *argv[])
 
 #ifdef ZJS_LINUX_BUILD
     zjs_free(script);
+    init_process(js_args);
```

Comment:
We should probably make this it's own module, conditional based on usage w/ analyze, like other things. Otherwise it's just going to be overhead. Particularly when we don't really have a use case. :)

So perhaps split that out here for a separate PR?